test/CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ if (NOT ${SPIRV_SKIP_EXECUTABLES})
       ${CMAKE_CURRENT_SOURCE_DIR}/ImmediateInt.cpp
       ${CMAKE_CURRENT_SOURCE_DIR}/LibspirvMacros.cpp
       ${CMAKE_CURRENT_SOURCE_DIR}/NamedId.cpp
+      ${CMAKE_CURRENT_SOURCE_DIR}/NameMapper.cpp
```

Comment:
Hmm, I'd suggest we avoid putting more stuff into the giant UnitSPIRV binary. It's already way too large, and if something goes wrong, it will output pages of pages of errors.

I won't bother for this time. I'll split it into multiple smaller ones later.